### PR TITLE
Added Amazon QLDB pack

### PR DESCRIPTION
**Pull request by EmilHammarstrand**
# Summary

Added dashboard images.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### CODE REVIEW

**Comment by jpvajda** [comment]
This is failing a PR check @EmilHammarstrand could you resolve? - Thank you! 

```
 Error: quickstarts/aws/aws-qldb/dashboards/qldb.json
	 '#/not' must NOT be valid
	 '#/additionalProperties' must NOT have additional properties
```

File: quickstarts/aws/aws-qldb/config.yml [hunk 1]
```diff
@@ -15,14 +15,15 @@ description: |-
 
   ### More info
 
-  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-qldb-monitoring-integration) to learn more about New Relic monitoring for AWS QLDB. 
+  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-qldb-monitoring-integration) to learn more about New Relic monitoring for AWS QLDB.
```
Comment (jpvajda) [inline quickstarts/aws/aws-qldb/config.yml:18]: @bradleycamacho any suggestions on this description?

Comment (jpvajda) [reply quickstarts/aws/aws-qldb/config.yml:18]: I'd probably recommend including some details of how we describe this on the docs page

New Relic offers an integration for reporting your Amazon Quantum Ledger Database (QLDB) data to New Relic. This document explains how to activate this integration and describes the data that can be reported.

**Review by jpvajda — CHANGES_REQUESTED** [review]

**Comment by EmilHammarstrand** [comment]
@jpvajda All checks has passed the pipeline now!

**Comment by jpvajda** [comment]
@EmilHammarstrand Thanks! There is a still my  description content question to resolve, just see the comment I made.

**Review by jpvajda — APPROVED** [review]

**Pull request merged**